exercises/twelve-days/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "xecmascript",
+  "version": "0.0.0",
+  "description": "Exercism exercises in ECMAScript 6.",
+  "author": "Aakash Mallik",
```

Comment:
FYI this will get overwritten with `"author": "Katrina Owen"` as the Makefile copies `package.json` from the top level.